Maestro/src/Microsoft.DotNet.Maestro/Services/VsoService.cs

```diff
@@ -16,18 +16,21 @@ namespace Microsoft.DotNet.Maestro.Services
 {
     public class VsoService
     {
-        private static HttpClient client = new HttpClient();
+        private HttpClient client;
```

Comment:
This is generally a bad change.  HttpClient instances shouldn't be created and then thrown away.

See [https://docs.microsoft.com/en-us/aspnet/web-api/overview/advanced/calling-a-web-api-from-a-net-client#create-and-initialize-httpclient](https://docs.microsoft.com/en-us/aspnet/web-api/overview/advanced/calling-a-web-api-from-a-net-client#create-and-initialize-httpclient)